polkadot-parachains/integritee-kusama/src/lib.rs

```diff
@@ -1104,12 +1104,7 @@ pub type SignedBlock = generic::SignedBlock<Block>;
 pub type BlockId = generic::BlockId<Block>;
 
 /// Migrations to apply on runtime upgrade.
-pub type Migrations = (
-	migrations::scheduler::v4::PurgeV4Agenda<Runtime>,
-	migrations::collator_selection_init::v0::InitInvulnerables<Runtime>,
-	cumulus_pallet_xcmp_queue::migration::v4::MigrationToV4<Runtime>,
-	cumulus_pallet_xcmp_queue::migration::v5::MigrateV4ToV5<Runtime>,
-);
+pub type Migrations = (migrations::scheduler::v4::PurgeV4Agenda<Runtime>,);
```

Comment:
Here I am not sure, I guess this has already been applied on the last runtime upgrade, but as this purge does not set a new version, it applies it again. What was the reason for introducing this again?

Reply:
should be removed even if it currently has no effect as nothing is scheduled. Context: https://github.com/integritee-network/parachain/issues/288